lib/matplotlib/backend_bases.py

```diff
@@ -2735,10 +2735,7 @@ def notify_axes_change(fig):
             if self.toolmanager is None and self.toolbar is not None:
                 self.toolbar.update()
 
-    @cbook.deprecated("3.3")
-    @property
-    def statusbar(self):
-        return None
+    statusbar = cbook.deprecated("3.3")(property(lambda self: None))
```

Comment:
Optionally, moving this to the top of the class where the class variables would normally go would probably make it easier to not page past it when scrolling quickly "looking" for "def ...".